runtimes/kestrel/src/lib.rs

```diff
@@ -156,16 +156,14 @@ impl frame_system::Config for Runtime {
 	/// The lookup mechanism to get account ID from whatever is passed in
 	/// dispatchers.
 	type Lookup = AccountIdLookup<AccountId, ()>;
-	/// The index type for storing how many extrinsics an account has signed.
-	type Index = Index;
-	/// The index type for blocks.
-	type BlockNumber = BlockNumber;
 	/// The type for hashing blocks and tries.
 	type Hash = Hash;
 	/// The hashing algorithm used.
 	type Hashing = BlakeTwo256;
-	/// The header type.
-	type Header = generic::Header<BlockNumber, BlakeTwo256>;
+	/// The block type as expected in this runtime
+	type Block = Block;
+	/// The nonce type for storing how many extrinsics an account has signed.
+	type Nonce = runtime_common::Nonce;
```

Comment:
Same here.

Reply:
updated in: [94b4a70](https://github.com/KILTprotocol/kilt-node/pull/559/commits/94b4a70489d1a4fa1503e9729089c2036be4ce4d)